compiler/luci/service/include/luci/Service/CircleShapeInference.h

```diff
@@ -165,6 +165,7 @@ class Algorithm final : public luci::CircleNodeVisitor<loco::TensorShape>
   // loco::TensorShape visit(const luci::CircleBCQFullyConnected *node) final;
   // loco::TensorShape visit(const luci::CircleBCQGather *node) final;
   // loco::TensorShape visit(const luci::CircleInstanceNorm *node) final;
+  // loco::TensorShape visit(const luci::CircleRoPE *node) final;
```

Comment:
I'm curious about why these comments are left, 
In case of RmsNorm, I just skipped because it is comment.

Reply:
thank you for your comment.
I also confirmed that it was deleted from rmsnorm.
so I was thinking about whether to delete or comment, but RoPE left it to express explicitly that it does not support shape interference.

@seanshpark 
am I right? and please give us your opinion on what to do.

Reply:
this comment is to remember this Op is needed to implement shape inference here.
this class is for dynamic shape inference.
someday we may need to implement here, but in the mean time current implementation in `CircleShapeInferenceRule.cpp` is OK.

Reply:
@seockho-kim 
i would like to keep the current code by referring to seanshpark's opinion,
please review it.